README.md

```diff
@@ -1 +1,49 @@
-# My_portofolio
\ No newline at end of file
+# My-portfolio
+This project is a very basic website that shows my portfolio in using HTML, CSS wihtout using JS
+
+## Built With
+
+- CSS
+- HTML
+
+## Live Demo
+
+[Live Demo Link](https:///)
+
+
+## Getting Started
+
+**To use this repositpry you will need to Clone this repository to your machine. A .gitignore file is already created to ignore any file that is irrelevant, node_modules file. A linter is already set up to help you check for errors in your codes**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+**Tap on the code option in the repo, then you copy the link to the repo provided**
+### Setup
+**Open your git bash or your IDE terminal then use the git command to clone it to your machine**
+### Install
+**The folder will then be installed in your machine.**
+### Usage
+**You can edit codes and make changes, dont forget to give credits to me**
+### Run tests
+**Linters are already setup in the repo for error checks**
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [bertinkambale](https://github.com/bertinkambale)
+- Gmail: [bertinkambale](bertinkambale@gmail.com)
+
+
+Contributions, issues, and feature requests are welcome !
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed
```

Comment:
- [OPTIONAL] At this moment, the link to the MIT License is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your repo.  I marked it as optional because it was not mentioned by the previous reviewer, but it is highly recommended you set it up.